data/card_definitions.json

```diff
@@ -9300,7 +9300,318 @@
 		},
 		"NOTEEND": "NAGO CARDS END"
 	},
-
+	{
+		"NOTE": "NANASE CARDS START",
+		"id": "nanase_atmosphereoftheaether",
+		"type": "ultra",
+		"display_name": "Atmosphere of the Aether",
+		"force_cost": 0,
+		"gauge_cost": 3,
+		"range_min": 1,
+		"range_max": 2,
+		"speed": 6,
+		"power": 4,
+		"armor": 0,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "hit",
+				"effect_type": "push",
+				"amount": 3,
+				"and": {
+					"effect_type": "draw",
+					"amount": 2
+				}
+			},
+		],
+		"boost": {
+			"boost_type": "continuous",
+			"force_cost": 0,
+			"cancel_cost": -1,
+			"display_name": "Zephyr",
+			"effects": [
+				{
+					"timing": "during_strike",
+					"effect_type": "speedup",
+					"amount": 2,
+					"and": {
+						"effect_type": "powerup_opponent",
+						"amount": 3	
+					}
+				},
+			]
+		}
+	},
+	{
+		"id": "nanase_lumiereofthedawn",
+		"type": "ultra",
+		"display_name": "Lumiere of the Dawn",
+		"force_cost": 0,
+		"gauge_cost": 3,
+		"range_min": 1,
+		"range_max": 1,
+		"speed": 4,
+		"power": 9,
+		"armor": 6,
+		"guard": 0,
+		"effects": [
+			{
+				"timing": "before",
+				"condition": "speed_greater_than",
+				"condition_amount": "OPPONENT_SPEED",
+				"effect_type": "atack_does_not_hit",
+				"and": {
+					"effect_type": "lose_all_armor"
+					}
```

Comment:
weird spacing showing up, mixing tabs/spaces? try to stick with just tabs for indents